docs/5.2. Transport - Websocket.md

```diff
@@ -108,14 +119,26 @@ A subscription command will only get sent out as a consequence of an NMOS Connec
 
 ### 4.1 Heartbeats
 
-Upon connection, the client is required to report its health every 60 seconds in order to maintain its session and subscription. This is similar to the behaviour required by the registry for nodes in IS-04 specification.
+Upon connection, the client is required to report its health every 10 seconds in order to maintain its session and subscription. This is similar to the behaviour required by the registry for nodes in IS-04 specification. Every `health` command should be followed by a `health` response (see [Message types](2.0.%20Message_types.md)).
```

Comment:
As Peter mentioned in his comment, 5 is the default for IS-04 (but can be changed). Should we implement the same defaults?